src/index.ts

```diff
@@ -1,32 +1,28 @@
-// this import must be called before the first import of tsyring
+/* eslint-disable import/first */
+// this import must be called before the first import of tsyringe
 import 'reflect-metadata';
 import { createServer } from 'http';
+import { createTerminus, HealthCheck } from '@godaddy/terminus';
 import { Logger } from '@map-colonies/js-logger';
-import { createTerminus } from '@godaddy/terminus';
-import { container } from 'tsyringe';
 import config from 'config';
 import { DEFAULT_SERVER_PORT, SERVICES } from './common/constants';
+
 import { getApp } from './app';
-import { ConnectionManager } from './DAL/connectionManager';
 
 const port: number = config.get<number>('server.port') || DEFAULT_SERVER_PORT;
 
-const app = getApp();
+void getApp()
```

Comment:
is it possible to add log here of - "starting app"

Reply:
what about line 20?